src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImage.groovy

```diff
@@ -94,6 +94,10 @@ class DockerBuildImage extends AbstractDockerRemoteApiTask implements RegistryCr
     @Optional
     final SetProperty<String> cacheFrom = project.objects.setProperty(String)
 
+    @Input
+    @Optional
+    final Property<String> target = project.objects.property(String)
```

Comment:
Would you mind add some Javadocs and a `@since` annotation with the proposed version?

Reply:
Java Doc added.